app/views/spree/admin/transaction_sync_batches/show.html.erb

```diff
@@ -22,6 +23,7 @@
         <td><%= link_to log.order.number, spree.edit_admin_order_path(log.order) %></td>
         <td><%= log.order_transaction&.transaction_id || "-" %></td>
         <td><%= log.status.capitalize %></td>
+        <td style="white-space: pre-wrap; width: 150px;"><%= log.error_message %></td>
```

Comment:
Thoughts on doing `.truncate(255)` or something along those lines here. Not sure if this could contain a stack trace, but if it does it'll be a very long and narrow column with this preset width.

Reply:
We can ignore this for now since this won't include stack trace.